server.js

```diff
@@ -80,7 +86,48 @@ app.post('/game', validate_create_game, function(req, res) {
     console_log("game {0} created by {1}".format(game.id, game.player1.id));
 });
 
+
 // GET game
+app.get('/setup', function(req, res, next) {
```

Comment:
@karadouville What is this endpoint used for? Can you just use `post /game` and `post /game/:id/join`? Seems strange to use `get` for creating a new game.

Reply:
@Austinpayne  this endpoint is hit by the login page, I guess it's part of the UI api, rather than the REST api. You're totally right, It is a duplication of /game and /game:id/join, something that I was planning on fixing down the line. I was just trying to crank something out and I needed a different response.  I'm working on extracting the logic from creating a game and joining a game and getting rid of the get setup.

Reply:
Yeah, just seems to be blurring the lines a lot. I'm fine with it, just wasn't sure what it is for. Where we are mixing the backend with frontend we should at least put it in a different spot and have some comment header marking it as the "UI" section.